types/jasmine/index.d.ts

```diff
@@ -435,59 +482,93 @@ declare namespace jasmine {
     }
 
     interface Env {
+        addAsyncMatchers: typeof addAsyncMatchers;
```

Comment:
This value isn't documented at https://jasmine.github.io/api/edge/Env

It seems to not be part of the public API. Is this intentional?

Reply:
None of the global or static function like [`addAsyncMatchers`](https://jasmine.github.io/api/edge/jasmine.html#.addAsyncMatchers) are documented on `Env` but some of them were already in the types so I wasn't sure If I should add all of them.

All of those functions are public.

Reply:
I would err on the side of not adding them, unless the project you are working on needs them. But, I'll approve this and let you decide.

@sgravrock Any chance you can comment on this, if you see this?